crates/turborepo-scm/src/git.rs

```diff
@@ -139,51 +186,53 @@ fn reanchor_path_from_git_root_to_turbo_root(
 pub fn previous_content(
     git_root: PathBuf,
     from_commit: &str,
-    file_path: PathBuf,
+    file_path: String,
 ) -> Result<Vec<u8>, Error> {
     // If git root is not absolute, we error.
     let git_root = AbsoluteSystemPathBuf::try_from(git_root)?;
+    let scm = SCM::new(&git_root);
 
     // However for file path we handle both absolute and relative paths
     // Note that we assume any relative file path is relative to the git root
-    let anchored_file_path = if file_path.is_absolute() {
-        let absolute_file_path = AbsoluteSystemPathBuf::try_from(file_path)?;
-        git_root.anchor(absolute_file_path)?
-    } else {
-        file_path.as_path().try_into()?
-    };
-
-    let git_binary = which("git")?;
-    let mut command = Command::new(git_binary);
-    let command = command
-        .arg("show")
-        .arg(format!("{}:{}", from_commit, anchored_file_path.as_str()))
-        .current_dir(&git_root);
-
-    let output = command.output()?;
-    if output.status.success() {
-        Ok(output.stdout)
-    } else {
-        Err(Error::Git(
-            String::from_utf8_lossy(&output.stderr).to_string(),
-            Backtrace::capture(),
-        ))
-    }
+    // FIXME: this is probably wrong. We should know the path to the lockfile
+    // exactly
+    let absolute_file_path = AbsoluteSystemPathBuf::from_unknown(&git_root, &file_path);
+
+    // let anchored_file_path = if file_path.is_absolute() {
```

Comment:
Should we delete this instead of commenting out?

Reply:
Yup